templates/ui-package/README.md

```diff
@@ -61,3 +61,10 @@ For adding env vars , you can use the .env file. For more information check
 To get more help on the shell-ui and lsc CLI go check out the
 [Shell CLI README](https://github.com/angular/shell-ui/blob/master/README.md).
 [LSC CLI README](https://github.com/angular/lsc/blob/master/README.md).
+
+## Cypress Configuration
+- `NPM_TOKEN`: An npm install token with access to the organization private repos.
```

Comment:
Maybe it could be noted that it can be an NPM token set with "--readonly" permissions